src/pages/foodSelectPage/navbar/Navbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import styled from "styled-components";
+import { ReactComponent as Icon } from "../../../icon/chevron-left.svg";
+
+const Navbar = () => {
+  return (
+    <NavbarWrapper>
+      <BackButton>
+        <Icon />
+      </BackButton>
+      <div>
+        <Title>취향 분석</Title>
+      </div>
+    </NavbarWrapper>
+  );
+};
+
+export default Navbar;
```

Comment:
Food Select Header로 네이밍 해주는 것은 어떨까요?  더 좋은 네이밍이 있으면 좋을 것 같아요 
추후에 Nav bar 헤더가 생긴다고 가정하면 네이밍이 겹칠 것 같습니다. 
```suggestion
export default FoodSelectHeader;
```